bazel-java-deps.bzl

```diff
@@ -161,7 +167,7 @@ def install_java_deps():
             "com.fasterxml.jackson.core:jackson-databind:2.12.0",
         ],
         fetch_sources = True,
-        maven_install_json = "@com_github_digital_asset_daml//:maven_install.json",
+        maven_install_json = "@com_github_digital_asset_daml//:maven_install_{}.json".format(scala_major_version),
```

Comment:
Is there any change that needs documenting when pinning the dependency graph?

Reply:
Good point, updated.